src/stylesheets/basic.css

```
@@ -220,6 +220,127 @@ main {
 }
 
 
+/* SEARCH SECTION */
+
+.list-search-container {
+  width: 1080px;
+  min-height: 32px;
+  margin-top: 10px;
+  margin-bottom: 5px;
+  padding: 10px;
+}
+
+.list-search-section {
+  display: flex;
+}
+
+.list-text-input {
+  background-color: rgba(255, 255, 255, 0.3);
+  border-style: none;
+  border-radius: 5px;
+  padding: 5px;
+  padding-left: 10px;
+  font-family: Quicksand;
+  font-weight: 300;
+  font-size: 15px;
+  color: white;
+}
+
+.list-text-input::placeholder {
+  color: white;
+  opacity: 70%;
+}
+
+.list-search-bar {
+  width: 1000em;
+}
+
+.list-search-content-toggle,
+.list-search-content-toggle img {
+  height: 32px;
+  cursor: pointer;
+}
+
+.list-search-content-toggle-button {
+  display: none;
+}
+
+.list-search-content-toggle label {
+  opacity: 50%;
+  transition: all .2s ease;
+}
+
+.list-search-content-toggle-button:checked + label {
+  opacity: 100%;
+}
+
+.list-search-filter-button,
+.list-search-sort-button {
+  margin-left: 10px;
+  padding-left: 6px;
+  padding-right: 6px;
+  border-radius: 5px;
+  background-color: rgba(255, 255, 255, 0.3);
+  transition: all .2s ease;
+}
+
+.list-search-filter-button svg,
+.list-search-sort-button svg {
+  height: 32px;
+  width: 20px!important;
+}
+
+.list-search-filter,
+.list-search-sort {
+  margin-top: 20px;
+}
+
+.list-search-content-title {
+  width: 1080px;
+  border-top-color: rgba(255, 255, 255, 0.4);
+  border-top-width: 1px;
+  border-top-style: solid;
+  padding-top: 10px;
+  font-size: 1.4em;
+  font-weight: 700;
+}
```

```suggestion
  margin-bottom: 5px;
}
```
i think it'd be nice to have a margin-bottom here

---

fixed